labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <title>Chirpus Inc.</title>
+    <link href="mobilefirststyle.css" rel="stylesheet" type="text/css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+  </head>
+  <body>
+    
+      <h3>chirpus inc.</h3>
```

Comment:
Keep your indentation consistent, please. In your editor, you can use Ctrl+shift+p and type 'indent' to search for a command that will reindent the whole file.